BricksAndHearts.UnitTests/ControllerTests/Admin/AdminControllerTests.cs

```diff
@@ -37,7 +37,7 @@ public async void LandlordList_WhenCalled_CallsGetUnapprovedLandlordsAndReturnsL
         var result = await _underTest.LandlordList() as ViewResult;
 
         // Assert
-        A.CallTo(() => adminService.GetUnapprovedLandlords()).MustHaveHappened();
+        A.CallTo(() => adminService.GetLandlordDisplayList("")).MustHaveHappened();

```

Comment:
You could mock this out above in the arrange section and make it return a specific list, then assert that the view model contains the correct data

Reply:
Is there an advantage to doing it that way? I might be misunderstanding, but it seems as though that would only confirm that the fake service is called and that the return from that service is registered by the controller - both parts of which are checked by the current test  - but would require a lot more set-up.

Reply:
It can be useful to check that the data returned from the service is correctly mapped to the view model.